src/Features/Core/Portable/PopulateSwitch/PopulateSwitchHelpers.cs

```diff
@@ -0,0 +1,131 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis.Semantics;
+using Microsoft.CodeAnalysis.Shared.Utilities;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.PopulateSwitch
+{
+    internal static class PopulateSwitchHelpers
+    {
+        public const string MissingCases = nameof(MissingCases);
+        public const string MissingDefaultCase = nameof(MissingDefaultCase);
+
+        public static bool HasDefaultCase(ISwitchStatement switchStatement)
+        {
+            for (var index = switchStatement.Cases.Length - 1; index >= 0; index--)
+            {
+                if (HasDefaultCase(switchStatement.Cases[index]))
+                {
+                    return true;
+                }
+            }
+
+            return false;
+        }
+
+        private static bool HasDefaultCase(ISwitchCase switchCase)
+        {
+            foreach (var clause in switchCase.Clauses)
+            {
+                if (clause.CaseKind == CaseKind.Default)
+                {
+                    return true;
+                }
+            }
+
+            return false;
+        }
+
+        public static ICollection<ISymbol> GetMissingEnumMembers(ISwitchStatement switchStatement)
```

Comment:
Would be really impressive if this supported the "completionlist tag" feature (where the IDE will act like static members of a non-enum type are enum members).